host-requirements.txt

```diff
@@ -26,8 +26,15 @@ tensorflow-gpu==1.15.0
 
 ### Begin Pytorch required libraries
 pytorch==1.7.0
-torchvision==0.8.0
+torchvision==0.7.0
```

Comment:
happened to glance at this PR and was curious why the change to torchvision version? For reference, corresponding PyTorch/torchvision versions are listed in this [table](https://pypi.org/project/torchvision/)

Reply:
I was curious, too; thanks for the reference document. There was a pip version error when running `--no-docker` which may not be the proper consumer of `host-requirements.txt`. I am reverting that version regression and will better explore what the issue was.

Reply:
I believe the packages added in 4e21bcd are only needed for the ASR scenario. Perhaps we should add a comment making this clear

Reply:
Probably should, but let's keep that out of this PR.